library/src/scala/quoted/Expr.scala

```diff
@@ -52,6 +52,16 @@ object Expr {
   def unapply[T](x: Expr[T])(using FromExpr[T])(using Quotes): Option[T] =
     scala.Predef.summon[FromExpr[T]].unapply(x)
 
+  /** Creates an expression that will construct a function with one parameter
+   *
+   *  Transforms the expression
+   *    `a => r` where `a: Expr[A]` and `r: Expr[R]`
+   *  to an expression equivalent to
+   *    `'{ $a => $r }` typed as an `Expr[A => R]`
```

Comment:
As you're adding a new method to stdlib's API you should mark it as`@experimental`